assets/scripts/components/scroll-spy.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var $ = require('jquery');
+var $nav = $('.js-sticky-nav');
+var originalNavigationHeight = $nav.outerHeight(true);
+
+function calculateSections ( $links ) {
+  // Get all links & sections and serialize them
+  //
+  return [].map.call( $links, function (el) {
+    var $link = $(el);
+    var $section = $($link.attr('href'));
+    var topOffset = $section.position().top - originalNavigationHeight;
```

Comment:
Make sure that the sticky navigation's height is taken into account for the `topOffset` of all sections.